web/src/lib/components/pages/record.tsx

```diff
@@ -117,6 +131,19 @@ export default class RecordPage extends Component<RecordProps, RecordState> {
 
     this.props.onRecordStop && this.props.onRecordStop();
 
+    const error = this.checkRecording();
+    if (error) {
+      // Remove the invalid recording to go back.
```

Comment:
For now, let's only console.log the error. We will wait until we a visual display to the user before we disallow recordings based on this validation.